src/components/rulesActionsList/ruleAndActionsConfig.js

```diff
@@ -9,6 +9,8 @@ import lang from "../../common/lang";
 
 export const LastTriggerDefaultWidth = 310;
 
+export const DEFAULT_RULES_GRID_PAGE_SIZE = 50;
```

Comment:
why is this constant all upper case, while the previous one is PascalCased? we should fix the inconsistencies.  Also please add a comment describing how the value (50 in this case) is calculated, otherwise it will be pretty much impossible to maintain it

Reply:
Sure. I will fix the casing issue but the 50 rows per page value is given by Caitlin.

Reply:
could you ask Caitlin how the constants were calculated?

also, this "50" is it the same of Config.DEVICES_RULESGRID_ROW, and could we reuse the Config value?